client/scss/components/_donate-form.scss

```
@@ -0,0 +1,29 @@
+.donate-form {
+  position: relative;
+
+  max-width: 45rem;
+
+  border: 1px solid $black;
+
+  input[type='text'] {
+    text-align: right;
+  }
+
+  label {
+    text-align: center;
+  }
+
+  .radio-input {
+    justify-content: center;
+  }
```

```suggestion
    .option .option-label .option-label-text {
      min-width: 4.5rem
    }
  }
```
Something like this should prevent shuffling between the amount tiers when changing currency. There may be a more elegant solution.